.travis.yml

```diff
@@ -5,12 +5,10 @@ python:
 install:
   - pip install tox
 
-script:
-  - python run_tests.py
-
 env:
   - TOXENV=flake8
-  - TOXENV=py27
+  - TOXENV=py27-django111
+  - TOXENV=py3-django2
```

Comment:
So `py3` would run _any_ Python 3 available, right? For this XBlock I think it's good.